player/lua/select.lua

```diff
@@ -604,6 +604,8 @@ mp.add_key_binding(nil, "menu", function ()
     end
 
     local menu = {
+        {"Open file(s)", "loadfiles", true},
```

Comment:
To be honest, I am still confused by this commit. select.lua is a script with internal ui. Currently, the existing menu items are used to open internal ui for selection. Why is adding a command to call out the dialog box of the external GUI in the menu that draws the internal ui? This is very fragmented. I tend to consider implementing internal file browsing and loading functions in the select.lua script later, and it is not necessary to be involved in this pr

Reply:
This is command menu as it stands now. There are commands that are not select and this is one of the useful ones, to open a file.

Reply:
> This is command menu as it stands now. There are commands that are not select and this is one of the useful ones, to open a file.

Yes, I just noticed that there are several commands in the menu that are not provided internally by that script. It might make sense to extend it further